validphys2/src/validphys/results.py

```diff
@@ -538,14 +538,11 @@ def procs_corrmat(procs_covmat):
 def results(dataset: DataSetSpec, pdf: PDF, covariance_matrix, sqrt_covmat):
     """Tuple of data and theory results for a single pdf. The data will have an associated
     covariance matrix, which can include a contribution from the theory covariance matrix which
-    is constructed from scale variation. The inclusion of this covariance matrix by default is used
-    where available, however this behaviour can be modified with the flag `use_theorycovmat`.
+    is constructed from scale variation.
```

Comment:
So this docstring was not true?

But the part of `The inclusion of this covariance matrix by default is used` is still true isn't it?

Reply:
I didn't trace back how `covariance_matrix` and `sqrt_covmat` are produced and whether they include the theory covmat or not, but at least the comment about `use_theorycovmat` was wrong